airflow-core/src/airflow/ui/src/i18n/README.md

```diff
@@ -17,8 +17,188 @@
+
+## 6. Procedures
+
+### 6.1. Approval process of a translation owner
+
+- The designated code owner should post a thread in the dev. list to request for approval of the translation owner(s) for a supported translation.:
+    - Approvals of a translation owner who is also the code owner can be done by a lazy consensus.
+    - Approvals of translation owners who are non-committers require at least one binding vote of at least 1 PMC member, and no objections from committers/PMC.
+- Within the thread, the code owner should demonstrate that the translation owner is suitable for the role, according to the requirements in section 4.2.
+
+### 6.2. Approval process of a new translation
+
+The following steps outline the process for approving a new translation:
+> [!WARNING]
+> Before merging the PR, the committer should validate that files are in sync with the default language (English).
+
+- Creating a PR to add a new translation to the codebase ([see example](https://github.com/apache/airflow/pull/51258/files)), according to the standard and guidelines, which includes:
+    - The translation files in the `airflow-core/src/airflow/ui/src/i18n/locales/<LOCALE_CODE>` directory, where `<LOCALE_CODE>` is the code of the language (e.g., `fr` for French).
+    - Making the required modifications in `airflow-core/src/airflow/ui/src/i18n/config.ts` ([see example](https://github.com/apache/airflow/pull/51258/files#diff-bfb4d5fafd26d206fb4a545a41ba303f33d15a479d21e0a726fd743bdf9717ff)) (should be done by the code owner).
+    - Updating the `.github/CODEOWNERS` file to include the code owner(s).
+      > [!NOTE]
+      > When assigning a translation sponsor, there should be a comment in the `.github/CODEOWNERS` indicating who the translation owner(s) and the translation sponsor are.
+- Apply procedure 6.1. to approve the translation owner, while announcing the introduction of the new language (including a link to the PR).
+- Only after the steps above are completed, the PR for the new translation could be merged (by the requirements in section 4.7).
+
+### 6.3. Translation conflict resolution
+
+When a conflict arises in a translation-related PR, the following steps will be taken in order:
+
+- The involved parties should first try to reach a consensus through discussion in the PR.
+- If no consensus is reached, a translation owner may decide the outcome.
+- If multiple translation owners are involved and cannot reach consensus, the code owner will decide. If the code owner is sponsored,
+they should base their decision on a neutral source (e.g., a third-party opinion, translation tool, or LLM such as ChatGPT or Claude).
+- If the conflict is between code owners, a PMC member will be involved to resolve the conflict.
+
+### 6.4. Relinquishing translation/code ownership
+
+- When a translation owner asks to relinquish their role, or becomes inactive, and there are no other translation owners available, the code owner should follow these steps:
```

Comment:
What is a codeowner just goes passive and does not fulfill the role? Would it be acceptable that a different code owner also starts the process asking about status and maintenance?

Reply:
Perhaps "inactive" should come with an exact definition, i.e. along the lines of "at least 3 months of the target language not being updated after changes to source strings were introduced".

Reply:
I'll add a definition for inactivity (tl;dr - two consecutive releases with incomplete translations, or inactivity in Airflow for more than 12 months)